.github/workflows/ci.yml

```diff
@@ -209,7 +209,14 @@ jobs:
       fail-fast: false
       matrix:
         node-version: [12, 16]
-        mssql-version: [2017, 2019]
+        # 2014 (version 12) is the lowest support MS SQL version.
```

Comment:
Could we have all these comments in the dialects index.js file? I think it would fit better there and there is no real need to have all this information twice.

Reply:
Sure